frontend/views/tasks/create.php

```diff
@@ -1,126 +1,167 @@
 <?php
 
-use frontend\models\Categories;
+/**
+ * $categories = []
+ * $errors = []
+ ** @var $categories
+ * */
+
+
+use frontend\assets\CreateTaskDropZone;
 use yii\helpers\Html;
 use yii\widgets\ActiveForm;
+
+CreateTaskDropZone::register($this);
```

Comment:
Давай пока закомментируем, чтобы перетаксивание файлов не сабмитило форму
```suggestion
// CreateTaskDropZone::register($this);
```